### ISSUE TITLE
Allow auto-connect to control server in master mode

### ISSUE THREAD

**Issue by leonardder** [NVDARemote/NVDARemote#88]
It is now possible to connect to a remote or self hosted control server, but only in slave mode. It should be possible the other way around as well, being the master. This way, you can configure one NVDA instance as master and one instance as slave, connecting automatically.
